src/shared/Microsoft.Git.CredentialManager/Settings.cs

```diff
@@ -392,6 +398,30 @@ public bool IsCertificateVerificationEnabled
             }
         }
 
+        public string SslCaInfo
+        {
+            get
+            {
+                // First, verify that, if schannel is the HTTP backend, sslCaInfo must be explicitly enabled to be used
+                if (TryGetSetting(null, KnownGitCfg.Http.SectionName, KnownGitCfg.Http.SslBackend, out string backend) &&
```

Comment:
Just checking, does `http.sslBackend` not have an environment variable equivalent? Just curious; I thought most Git settings had both.

Reply:
As far as I can tell, it doesn't - it's not listed in its config entry in the [Git documentation](https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpsslBackend), and a quick search through the code didn't show a corresponding environment variable.

Reply:
Yeah, I just did a short search too and didn't find anything. Will resolve this comment.